src/main/java/org/sonar/plugins/csharp/RoslynProfileExporter.java

```diff
@@ -62,32 +62,32 @@ public RoslynProfileExporter(Settings settings, RulesDefinition[] rulesDefinitio
 
   public static List<PropertyDefinition> sonarLintRepositoryProperties() {
     return Arrays.asList(
-      PropertyDefinition.builder(pluginKeyPropertyKey(SONARLINT_PARTIAL_REPO_KEY))
+      PropertyDefinition.builder(pluginKeyPropertyKey(SONARANALYZER_PARTIAL_REPO_KEY))
         .defaultValue("csharp")
         .hidden()
         .build(),
-      PropertyDefinition.builder(pluginVersionPropertyKey(SONARLINT_PARTIAL_REPO_KEY))
-        .defaultValue(SONARLINT_NUGET_VERSION)
+      PropertyDefinition.builder(pluginVersionPropertyKey(SONARANALYZER_PARTIAL_REPO_KEY))
+        .defaultValue(SONARANALYZER_NUGET_VERSION)
         .hidden()
         .build(),
-      PropertyDefinition.builder(staticResourceNamePropertyKey(SONARLINT_PARTIAL_REPO_KEY))
-        .defaultValue("SonarLint.zip")
+      PropertyDefinition.builder(staticResourceNamePropertyKey(SONARANALYZER_PARTIAL_REPO_KEY))
+        .defaultValue("SonarAnalyzer.zip")
         .hidden()
         .build(),
-      PropertyDefinition.builder(analyzerIdPropertyKey(SONARLINT_PARTIAL_REPO_KEY))
-        .defaultValue("SonarLint.CSharp")
+      PropertyDefinition.builder(analyzerIdPropertyKey(SONARANALYZER_PARTIAL_REPO_KEY))
+        .defaultValue("SonarAnalyzer.CSharp")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SonarAnalyzer.CSharp" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)